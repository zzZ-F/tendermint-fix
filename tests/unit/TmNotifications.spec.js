import { mount } from "@vue/test-utils"
import htmlBeautify from "html-beautify"
import { TmNotifications } from "@/index"

describe("TmNotifications.vue", () => {
  let wrapper
  beforeEach(async () => {
    const notifications = [
      {
        body: "asdf"
      },
      {
        body: "sadf"
      }
    ]
    wrapper = mount(TmNotifications, {
      propsData: { notifications }
    })
  })

  it("has the expected html structure", () => {
    expect(htmlBeautify(wrapper.html())).toMatchSnapshot()
  })
})
