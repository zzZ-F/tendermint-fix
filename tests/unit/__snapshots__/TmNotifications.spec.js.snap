// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmNotifications.vue has the expected html structure 1`] = `
<div class="tm-notifications">
  <div class="tm-notification">
    <header>
      <!---->
      <!---->
      <menu>
        <!---->
        <i class="close material-icons">close</i>
      </menu>
    </header>
    <div class="body">
      asdf
    </div>
  </div>
  <div class="tm-notification">
    <header>
      <!---->
      <!---->
      <menu>
        <!---->
        <i class="close material-icons">close</i>
      </menu>
    </header>
    <div class="body">
      sadf
    </div>
  </div>
</div>
`;
